script/testing/oltpbench/README.md

```diff
@@ -0,0 +1,65 @@
+# OLTP Benchmark Testing
+This folder include the necessary files for running oltpbench testings.
+
+# OLTP benchmark types
+Currently we run the following OLTP benchmarks
```

Comment:
To make this documentation survive longer we should explain how to check which benchmarks are being run. I think we should pull the information from the config/README.md into this README.md

Reply:
I have moved the contents from the `config/README.md` to the overall oltpbench `README.md`